### PR TITLE
Fix rendering of emojis in roster message

### PR DESCRIPTION
**Pull request by xmadsen**
remove spaces in constants and update tests

### CODE REVIEW

**Comment by xmadsen** [comment]
before:


: astonished:  Villager | : simple_smile:  Alive | @Head of the Xander's Career Transition Team
: astonished:  Villager | : skull:  Dead | @green bean cas-sar-ole
: japanese_goblin:  Mafia | : skull:  Dead | @Oil Daddy
: astonished:  Villager | : simple_smile:  Alive | @the curse of chloe
: astonished:  Villager | : simple_smile:  Alive | @Retrac Nnugliani

**Comment by xmadsen** [comment]
after:

:astonished:  Villager | :simple_smile:  Alive | @Head of the Xander's Career Transition Team
:astonished:  Villager | :skull:  Dead | @green bean cas-sar-ole
:japanese_goblin:  Mafia    | :skull:  Dead | @Oil Daddy
:astonished:  Villager | :simple_smile:  Alive | @the curse of chloe
:astonished:  Villager | :simple_smile:  Alive | @Retrac Nnugliani

**Comment by xmadsen** [comment]
but i guess github doesn't like simple_smile or rendering multiple spaces in a row :(

File: util/game_message_builder.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def build_roster_message(gameState, isGameOver=False):
     roster_message = ''
     for p in gameState.players:
         if isGameOver:
-            roster_message += f'{role_emoji(p.role)}  {p.role.capitalize()} | '
+            roster_message += f'{role_emoji(p.role)}  {p.role.capitalize()} {"   " if p.role == Roles.MAFIA else ""}| '
```
Comment (jacksonal) [inline util/game_message_builder.py:36]: nice!

**Review by jacksonal — APPROVED** [review]

**Pull request merged**